Apps/WebClient/src/ClientApp/src/components/timeline/note.vue

```diff
@@ -238,6 +240,7 @@ export default class NoteTimelineComponent extends Vue {
     private dateString: string = new Date().toISOString().slice(0, 10);
     private detailsVisible = false;
     private hasErrors: boolean = false;
+    private errorDescription: string = "";
```

Comment:
naming suggestion: errorDescription --> errorDetails or errorMessage depending on if this field stores just a message to user or more details for trouble-shooting error. Ideally, an error should contain both message and details.

Reply:
This will probably go away once we change it to work with the card componetn